src/data/services/LicenseManagerApiService.js

```diff
@@ -8,6 +10,21 @@ class LicenseManagerApiService {
     const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/${subscriptionUUID}/licenses/assign/`;
     return apiClient.post(url, options, 'json');
   }
+
+  static fetchSubscriptions(options) {
+    const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/?${qs.stringify(options)}`;
+    return apiClient.get(url);
+  }
+
+  static fetchSubscriptionUsers(subscriptionUUID, options) {
+    const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/${subscriptionUUID}/licenses?${qs.stringify(options)}`;
```

Comment:
Reiterating from my earlier comment that we should have trailing slash `/` at the end of these API URLs, e.g., `/licenses/?...`.

Reply:
Sorry, I removed them in the previous commit by mistake, I am adding them now.